### PR TITLE
Feat: Support existing users and user schemas

### PR DESCRIPTION
**Pull request by notshof**

### CODE REVIEW

**Comment by notshof** [comment]
This is a fix/feature to support manually added custom schemas as well as users.

File: gsuite/resource_user.go [hunk 1]
```diff
@@ -439,15 +439,46 @@ func resourceUserCreate(d *schema.ResourceData, meta interface{}) error {
 	}
 	user.Name = userName
 
-	var createdUser *directory.User
 	var err error
+	var existingUsers *directory.Users
+	err = retry(func() error {
+	    existingUsers, err = config.directory.Users.List().Customer("my_customer").Query("email:" + user.PrimaryEmail).Do()
```
Comment (DeviaVir) [inline gsuite/resource_user.go]: Should `my_customer` be `config.CustomerId`?
https://github.com/DeviaVir/terraform-provider-gsuite/blob/2a2c0f26dd0e6107a4e55d235b95487de7c24548/gsuite/provider.go#L104-L109

Comment (DeviaVir) [reply gsuite/resource_user.go]: looks like https://github.com/DeviaVir/terraform-provider-gsuite/blob/458ff4433659c5098d588f36b211adbd99fd22de/gsuite/resource_user_schema.go using `MyCustomerId` should probably be updated as well.

Comment (notshof) [reply gsuite/resource_user.go]: Good catch, this has been resolved in both files.

Comment (DeviaVir) [reply gsuite/resource_user.go]: Thank you very much!

**Comment by DeviaVir** [comment]
I'll be building a release tomorrow that will include this PR. Thank you very much for working on this!

**Comment by DeviaVir** [comment]
https://github.com/DeviaVir/terraform-provider-gsuite/releases/tag/v0.1.35

**Pull request merged**